### DISCUSSION TITLE
Created migration guide from streamdeck_ui

### DISCUSSION THREAD

**Discussion by RemcoSchrijver** [streamdeck-linux-gui/streamdeck-linux-gui#34] [Show and tell]
I wrote up a small migration guide [here](https://writerit.nl/productivity/streamdeck/migrating-from-streamdeck_ui-to-streamdeck-linux-gui/), it is done with ubuntu as a base but everything except for systemd should be transferable to other OS installations.

Might be interesting to post this in other issues as well as maybe making this part of the documentation?

**Comment by Danie10** [comment]
Thanks, now I know where the default settings file is. I was looking under ~/.config or ~/.local the whole time.

The image seems to not be loading for me, though.

I'm using AUR so in my case it was just the normal upgrade which ran through fine with no intervention.

**Reply by RemcoSchrijver** [reply]
I think the aur repo is [not yet updated](https://github.com/timothycrosley/streamdeck-ui/issues/357#issuecomment-1654606072).

So for now I think you might want to use pip/pipx to get the new version. Or wait until the AUR package is updated. If it is I'd like to know and I'll update my guide.

**Comment by Danie10** [comment]
@RemcoSchrijver yes it was. There was a comment posted there, and since installing, mine now works with Pillow v10 and shows Devel. For me, it was just a simple update.

**Comment by coolapso** [comment]
thank you @RemcoSchrijver 

Also your article just highlighted a bug :D the version showing as "Devel" was actually not supposed to and it should have been showing correctly.

I have pushed a fix to the Development branch already. 

For those wondering about the AUR package, there are currently 2 AUR packages but only one of them has been moved so far.
This is the one you should be using: https://aur.archlinux.org/packages/streamdeck-ui

**Reply by RemcoSchrijver** [reply]
Alright cool :) I updated the guide to reflect my experience being a bug as well as a section for user who installed using AUR!
